README.md

```diff
@@ -221,7 +221,65 @@ func main() {
 	fmt.Printf("Status after transition: %+v\n", issue.Fields.Status.Name)
 }
 ```
+### Get all the issues for JQL with Pagination
+Jira API has limit on maxResults it can return. YOu may have a usecase where you need to get all issues for given JQL.
+This example shows reference implementation of GetAllIssues function which does pagination on Jira API to get all the issues for given JQL
+```go
+package main
+
+import (
+	"fmt"
+
+	jira "github.com/andygrunwald/go-jira"
+)
 
+//GetAllIssues takes a jira client and returns all issues for given JQL
+func GetAllIssues(client *jira.Client, searchString string) ([]jira.Issue, error) {
+	last := 0
+	var issues []jira.Issue = nil
```

Comment:
[]jira.Issue will be nil automatically, this should just be `var issues []jira.Issue`